flow/project.py

```diff
@@ -265,8 +277,135 @@ def make_bundles(operations, size=None):
     return _make_bundles(operations, size)
 
 
+class aggregate(object):
+    """Decorator for operation functions that needs to be aggregated.
+
+    .. code-block:: python
+        example_aggregate = aggregate()
+
+        @example_aggregate
+        @FlowProject.operation
+        def foo(jobs):
+            return len(jobs)
+
+    :param grouper:
+        Information on how to aggregate jobs.
```

Comment:
Needs more documentation about the type of callable accepted. Should it take in a list or a variable number of positional parameters for instance. Does it need to return a list of lists or can it just yield lists or single jobs instead.